### PR TITLE
Determine and report test coverage also for PRs

### PR DESCRIPTION
**Pull request by marcjansen**
## Description

This PR suggests to determine and report the test coverage also for PRs, not oly for merges onto master.

## Pull request type

Please check the type of change your PR introduces:

- [x] Build related changes

## Do you introduce a breaking change?

- [x] No
## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [BSD 2-Clause License](https://github.com/geostyler/geostyler/blob/master/)
- [x] I have followed the [guidelines for contributing](https://github.com/geostyler/geostyler/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/geostyler/geostyler/blob/master/CODE_OF_CONDUCT.md)
- ~~[ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)~~ n.a.

### CODE REVIEW

File: .github/workflows/on-pull-request.yml [hunk 1]
```diff
@@ -33,3 +33,11 @@ jobs:
 
     - name: Build artifacts 🏗️
       run: npm run build
+
+    - name: Generate coverage 🧪
+      run: npm run test:ci
```
Comment (KaiVolland) [inline .github/workflows/on-pull-request.yml:38]: Coverage should already be created when running `npm run build` (`"prebuild": "npm run test && rimraf dist/** && rimraf build/**"`). Please check if you can avoid running the tests twice.

**Review by KaiVolland — COMMENTED** [review]
Very nice. I was just looking into this.

File: .github/workflows/on-pull-request.yml (hunk 1 above)
Comment (marcjansen) [reply .github/workflows/on-pull-request.yml:38]: even better, looking into this

**Comment by marcjansen** [comment]
Ping @KaiVolland updated as suggested, thanks for the review.

**Review by KaiVolland — APPROVED** [review]

**Comment by marcjansen** [comment]
Thanks, Kai.

**Pull request merged**